tensorflow/python/training/learning_rate_decay.py

```diff
@@ -19,7 +19,8 @@
 from __future__ import print_function
 
 from tensorflow.python.framework import ops
-from tensorflow.python.ops import math_ops
+from tensorflow.python.ops.math_ops import add, cast, div, exp, floor, mod, mul, neg
```

Comment:
We generally don't import this way: we import the module and fully qualify them.